lib/head.js

```diff
@@ -13,7 +13,7 @@ class Head extends React.Component {
 }
 
 export function defaultHead () {
-  return [<meta charSet='utf-8' className='next-head' />]
+  return [<meta charSet='utf-8' />]
```

Comment:
I don't think this change is needed

Reply:
@timneutkens Will this work anyway ?, If this is necessary I can take a look again.

Reply:
@timneutkens Well I meant in general, if you consider this PR as a fix to a real issue, I can take a look into it again, things may have changed in 11 months...

Reply:
With my comment in place it's the correct PR.